src/pages/my-courses/MyCourses.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react'
+import Tabs from '@mui/material/Tabs'
+import Box from '@mui/material/Box'
+
+import Tab from '~/components/tab/Tab'
+import PageWrapper from '~/components/page-wrapper/PageWrapper'
+
+const coursesData = [
+  {
+    id: 1,
+    title: 'Course1',
+    lessons: [
+      { id: 1, title: 'Lesson1' },
+      { id: 2, title: 'Lesson2' },
+      { id: 3, title: 'Lesson3' }
+    ]
```

Comment:
I guess it needs to move to a constant file.

Reply:
maybe no need for that file, as it will be removed later

Reply:
Or if it is mock data, call this as CoursesDataMock.